config/sync/migrate_plus.migration_group.cardinal_service_opportunity.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 742dc185-ba4f-4f67-8067-410bf16c5a4a
```

Comment:
having migration groups just adds extra steps to perform the migration. just set the migration group to `default`. Then you only have 2 configs instead of 4.